Dfe.Academies.Academisation.Service/Mappers/Legacy/ProjectAggregate/LegacyProjectServiceModelMapper.cs

```diff
@@ -1,17 +1,18 @@
-﻿using Dfe.Academies.Academisation.IDomain.ProjectAggregate;
+﻿using Dfe.Academies.Academisation.Domain.Core.ProjectAggregate;
+using Dfe.Academies.Academisation.IDomain.ProjectAggregate;
 using Dfe.Academies.Academisation.IService.ServiceModels.Legacy.ProjectAggregate;
+using User = Dfe.Academies.Academisation.IService.ServiceModels.Legacy.ProjectAggregate.User;
 
 namespace Dfe.Academies.Academisation.Service.Mappers.Legacy.ProjectAggregate;
 
 internal static class LegacyProjectServiceModelMapper
 {
 	internal static LegacyProjectServiceModel MapToServiceModel(this IProject project)
 	{
-		LegacyProjectServiceModel serviceModel = new(
-			project.Id,
-			project.Details.Urn
-			)
+		LegacyProjectServiceModel serviceModel = new()
 		{
+			Id = project.Id,
```

Comment:
I think these were deliberately in the constructor because they were required fields. Everything else is nullable.